alphatwirl/selection/modules/CountWeight.py

```diff
@@ -0,0 +1,81 @@
+import copy
+
+##__________________________________________________________________||
+N_KEYS = 3
+IDX_DEPTH = 0
+IDX_PASS = 3
+IDX_TOTAL = 4
+
+##__________________________________________________________________||
+class Count(object):
+    """ Similar to Count.py, but increments by the value in event.w 
+    instead of 1.
+    """
+
+    def __init__(self):
+        self._results = [ ]
+
+    def __repr__(self):
+        return '{}({!r})'.format(self.__class__.__name__, self._results)
+
+    def copy(self):
+        return copy.deepcopy(self)
+
+    def add(self, selection):
+        class_name = selection.__class__.__name__
+        selection_name = selection.name if hasattr(selection, 'name') and selection.name is not None else ''
+        depth = 1
+        pass_ = 0
+        total = 0
+        self._results.append([depth, class_name, selection_name, pass_, total])
+
+    def count(self, pass_, event):
+        for r, p in zip(self._results, pass_):
+            r[IDX_TOTAL] += event.w[0] # total
+            if p: r[IDX_PASS] += event.w[0] # pass
```

Comment:
Just to confirm, this `Count` class only increments the count by whatever is contained in `event.w` then?  It would be nice to be able to select the attribute that's used somewhere (presumably as an argument to `__init__` or through some setter, but we can always add that in in a future version, perhaps.